### ISSUE TITLE
[BUG] MCC won't run if Sentry is disabled

### ISSUE THREAD

**Issue by ObscenityIB** [MCCTeam/Minecraft-Console-Client#2779]
### Prerequisites

- [X] I made sure I am running the latest [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
- [X] I tried to [look for similar issues](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one
- [X] I have set `debugmessages=true` in config to diagnose my issue
- [X] I have redacted session tokens and passwords before attaching screenshots

### Minecraft Version

N/A

### Console Client Version

ca966a464c2262a7b51f6395fe901b12b832e7ee

### Expected Behavior

MCC to run with analytics disabled.

### Actual Behavior

MCC crashes if analytics are not enabled.

### Steps to Reproduce the bug

1. Edit default ini to set EnableSentry=false
2. Run MCC
3. Crash

### Anythings that could help diagnosing the bug

```txt
Minecraft Console Client v1.20.4 - for MC 1.4.6 to 1.20.4 - Github.com/MCCTeam
[MCC] [ScriptScheduler] Loaded task:
triggeronfirstlogin: False
 triggeronlogin: False
 triggerontime: True
 triggeroninterval: True
 timevalue: 14:00:00
 timeinterval: 53.4
 action: send /hello
[MCC] [ScriptScheduler] Loaded task:
triggeronfirstlogin: False
 triggeronlogin: True
 triggerontime: False
 triggeroninterval: False
 timevalue: 
 timeinterval: 56
 action: send /login pass
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at MinecraftClient.Program.Main(String[] args)
Aborted (core dumped)
```
```


### Device

Laptop

### Operating System

Linux

### Server Address (If applicable)

_No response_

**Comment by breadbyte** [comment]
Could not reproduce with the build from https://github.com/MCCTeam/Minecraft-Console-Client/releases/tag/20240713-273, on both Windows and Linux.

Did you build MCC from source?

**Comment by breadbyte** [comment]
Should be fixed in c69cdad. Thanks for the report!
